### PR TITLE
AP_Scripting: add to AHRS bindings and add Baro

### PR DESCRIPTION
**Pull request by IamPete1**
This work was funded by ARACE

This adds a few AHRS and baro bindings. Hopefully nothing too controversial, I had to add a helper function to AHRS due to the existing airspeed function using references rather than pointers.

### CODE REVIEW

**Comment by rmackay9** [comment]
I think it might be better to get the airspeed as a singleton instead of grabbing it from the ahrs.

**Comment by IamPete1** [comment]
@rmackay9 I want the estimated air-speed if no real airspeed is available, I think this is only possible via the ahrs?

**Comment by WickedShell** [comment]
The estimate is only available/remotely useful if it's a fly forward
vehicle, or has a sensor. That is a fairly important caveat...

**Comment by rmackay9** [comment]
re the caveat, I think the airspeed sensor is only ever connected to a plane so it shouldn't be an issue.  I've heard it be discussed for other vehicles but never actually seen it done in practice.

**Comment by IamPete1** [comment]
Rebased and moved is flying and flying time functions from AHRS to vehicle. If everyone is happy with these change i can squash the scripting changes

File: libraries/AP_AHRS/AP_AHRS.h [hunk 1]
```diff
@@ -285,6 +253,11 @@ class AP_AHRS
     // if we have an estimate
     virtual bool airspeed_estimate(float *airspeed_ret) const WARN_IF_UNUSED;
 
+    // airspeed helper used by scripting
+    bool get_airspeed(float &air_speed) const {
```
Comment (tridge) [inline libraries/AP_AHRS/AP_AHRS.h]: better to fix the airspeed_estimate() to take a reference

**Comment by IamPete1** [comment]
This now includes the moving of set/get/get_time flying from ahrs to vehicle and a update of airspeed_estimate to use pointers not references. Then adds the bindings.

File: libraries/AP_AHRS/AP_AHRS.h (hunk 1 above)
Comment (IamPete1) [reply libraries/AP_AHRS/AP_AHRS.h]: fixed

**Comment by rmackay9** [comment]
I'm happy with the Copter and Rover changes

**Comment by rmackay9** [comment]
@tridge says he's happy with the airspeed changes.

@WickedShell to review in the coming days..

**Pull request merged**